### PR TITLE
Do not expose 'cache-features' in the Mapconfig API

### PR DESCRIPTION
**Pull request by Algunenano**

### CODE REVIEW

File: test/unit/render_cache.test.js [hunk 1]
```diff
@@ -130,13 +130,13 @@ describe('render_cache', function() {
         });
     });
 
-    it('Works with cache-features = false in layer', function(done){
+    it('Does not work with cache-features = false in layer', function(done){
```
Comment (rafatower) [inline test/unit/render_cache.test.js]: I'd rephrase it to "works with cache-features = true by default".

Comment (Algunenano) [reply test/unit/render_cache.test.js]: I'll change it to `Ignores cache-features = false in layer`. The Default is a couple of test above

**Comment by rochoa** [comment]
Thank you, and sorry I didn't catch this change before.

File: test/unit/render_cache.test.js (hunk 1 above)
Comment (Algunenano) [reply test/unit/render_cache.test.js]: `Ignores cache-features = false in layer (Uses default = true)'` was clearer

**Comment by Algunenano** [comment]
Merging. I'll do a new release.

**Comment by rafatower** [comment]
My bad I didn't catch it despite the change in the map config doc, sorry  :(

**Comment by Algunenano** [comment]
Np, solved now in 4.3.3. I think this one could be merged now as it is
